server/market/market.go

```diff
@@ -1705,6 +1705,8 @@ func (m *Market) processReadyEpoch(epoch *readyEpoch, notifyChan chan<- *updateS
 			len(matches), len(partial), len(doneOK),
 			len(booked), len(unbooked), len(failed),
 		)
+	} else {
+		log.Tracef("No orders to match for market %v, epoch %d", m.marketInfo.Name, epoch.Epoch)
```

Comment:
Do you find this particularly helpful?  Even with TRC level, it's gonna be too spammy for my taste as at this stage we're running with TRC still